scripts/jenkins/cloud/ansible/bootstrap-vcloud-nodes.yml

```diff
@@ -27,16 +27,30 @@
   tasks:
     - block:
         - name: Ensure ardana SSH key on deployer
```

Comment:
If this task creates SSH keys for both Ardana and root, the task name should reflect that.